packages/apps/reputation-oracle/server/src/common/utils/cron.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { BACKOFF_INTERVAL_SECONDS } from '../constants';
+import { calculateBackoff } from './cron';
+
+describe('calculateBackoff', () => {
+  it('should return 2 minutes backoff for the first retry (retriesCount = 0)', () => {
+    const retriesCount = 0;
+    const result = calculateBackoff(retriesCount);
+    expect(result).toBe(BACKOFF_INTERVAL_SECONDS * 1000); // 2 minutes in milliseconds
```

Comment:
I think it would be enough to create a dummy constant that we will only use in these specific tests (`export const MOCK_BACKOFF_INTERVAL_SECONDS = 120;`), but we it will be also enough to hardcode it directly in the test.

Reply:
Dummy constant for tests is event better if we have multiple usages

Reply:
Okay, I will use it